felles/src/main/kotlin/no/nav/helsearbeidsgiver/felles/rapidsrivers/river/OpenRiver.kt

```diff
@@ -30,9 +30,7 @@ internal class OpenRiver(
         packet
             .toJson()
             .parseJson()
-            .toMap()
-            .filterValues { it !is JsonNull }
-            .let(messageHandler::lesOgHaandter)
+            .haandterMelding()
```

Comment:
Skal ikke denne lenger kalle på lesOgHaandter?

Reply:
Ah, riktig ser det nå 👍 Oppfører dette seg noe annerledes enn tidligere, eller er det kun en refaktorering? (Det ser ut som sistnevnte for meg 😄)

Reply:
Det skal oppføre seg helt likt. Jeg gjorde den endringen for at `OpenRiver` skulle kunne brukes med `PriObjectRiver`, som ikke bruker den fjernede `.toMap()`.